src/Servers/Kestrel/shared/HPackHeaderWriter.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
I think this can be removed?

Reply:
Removing it.

Reply:
I wondered if that was because of differences between Kestrel and http2cat.